apps/gitness/src/layouts/PullRequestSandboxLayout.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from 'react'
+import { NavLink, Outlet, useParams } from 'react-router-dom'
+import { Badge, Icon, Spacer } from '@harnessio/canary'
+import { SandboxLayout, PullRequestHeader } from '@harnessio/playground'
+import { TypesPullReq, useGetPullReqQuery } from '@harnessio/code-service-client'
+import { useGetRepoRef } from '../framework/hooks/useGetRepoPath'
+import { PathParams } from '../RouteDefinitions'
+
+const PullRequestSandboxLayout: React.FC = () => {
+  const [pullRequest, setPullRequest] = useState<TypesPullReq>()
+  const { pullRequestId } = useParams<PathParams>()
+  const repoRef = useGetRepoRef()
+  const prId = (pullRequestId && Number(pullRequestId)) || -1
+  const { data: { body: pullRequestData } = {}, isFetching } = useGetPullReqQuery({
+    repo_ref: repoRef,
+    pullreq_number: prId
+  })
+
+  useEffect(() => {
+    if (!isFetching && pullRequestData) {
+      setPullRequest(pullRequestData)
+    }
+  }, [pullRequestData, isFetching])
+
+  const baseClasses =
+    'inline-flex items-center justify-center px-3 py-1 px-4 items-center gap-2 bg-background hover:text-primary h-[36px] rounded-tl-md rounded-tr-md m-0'
+  const getLinkClasses = (isActive: boolean) => {
+    return `${baseClasses} ${isActive ? 'text-primary [&svg]:text-primary tabnav-active' : 'tabnav-inactive'}`
+  }
+  return (
+    <>
+      <SandboxLayout.Main hasHeader hasLeftPanel>
```

Comment:
Are you planning to remove reference to 'Sandbox' in the future, and simply call it Layout?

Reply:
+1

Reply:
Open to suggestions, can be done.